wire/messages.go

```diff
@@ -0,0 +1,175 @@
+package wire
+
+import (
+	"bytes"
+	"context"
+	"errors"
+	"fmt"
+
+	"github.com/libp2p/go-libp2p-core/host"
+	"github.com/libp2p/go-libp2p-core/peer"
+	"github.com/libp2p/go-libp2p-core/routing"
+
+	logging "github.com/ipfs/go-log"
+	record "github.com/libp2p/go-libp2p-record"
+	recpb "github.com/libp2p/go-libp2p-record/pb"
+	"github.com/multiformats/go-multihash"
+
+	"github.com/libp2p/go-libp2p-kad-dht/internal"
+	pb "github.com/libp2p/go-libp2p-kad-dht/pb"
```

Comment:
can we have this live in the same package as the generated protocol buffer? is there a reason to have it split into two?

Reply:
not particularly, I tend to think of the `pb` folder has just having codegen stuff, but we do some to have other things in there. Can move it though.